### DISCUSSION TITLE
Invalid SSL certificate for https://docs.modcluster.io/

### DISCUSSION THREAD

**Discussion by Loic-LeMeur** [modcluster/mod_cluster#574] [General]
Hello, 
It is no longer possible to access the documentation. It seems that this is due to a certificate problem. 
Would it be possible to access the documentation? 

Thank you for your feedback

**Comment by rhusar** [comment]
Thanks for the report @Loic-LeMeur, I will look into this.

**Reply by rhusar** [reply]
Just to add the main website is not accessible either https://modcluster.io/

**Reply by rhusar** [reply]
The problem is (as always) with the domain setup required by GitHub.

> Enforce HTTPS — Unavailable for your site because your domain is not properly configured to support HTTPS ([modcluster.io](https://modcluster.io/)) — [Troubleshooting custom domains](https://docs.github.com/articles/troubleshooting-custom-domains/#https-errors)

**Reply by rhusar** [reply]
All resolved now, with other fixes. Thanks for the report @Loic-LeMeur !

BTW anyone coming here needing to learn more about the setup I will document the reconfiguration done today into https://issues.redhat.com/browse/MODCLUSTER-748.

**Reply by Loic-LeMeur** [reply]
Thank you @rhusar . This documentation is greatly appreciated.
